noxconfig.py

```diff
@@ -20,6 +42,7 @@ class Config:
     doc: Path = Path(__file__).parent / "doc"
     version_file: Path = Path(__file__).parent / "exasol" / "toolbox" / "version.py"
     path_filters: Iterable[str] = ("dist", ".eggs", "venv", "metrics-schema")
+    plugins = [UpdateTemplates]
```

Comment:
I guess, we only need this in this repo

Reply:
Yes, other repos could define their own if needed.